README.md

```diff
@@ -50,35 +50,37 @@ pip install git+https://github.com/gridsingularity/d3a-api-client.git
 
 ## How to use the Client
 
-### Interacting with the CLI
-You can run one of the setup scripts residing in the `d3a_api_client/setups` folder, or you can 
-set the path for your scripts directory as list in the `d3a_api_client.constants` .
-In order to run a module you can use the following command
-```
-d3a-api-client run --setup {module_name}
-```
+### Interacting via CLI
+It helps to trigger your client script interact with your interested d3a's collaboration/CN conveniently via bash terminal.
```

Comment:
I dont see the information in this sentence.

Reply:
Any suggestions?, else will remove it then

Reply:
Yeah, remove it I would say.